pyro/contrib/minipyro.py

```diff
@@ -275,14 +264,14 @@ def __call__(self, params):
 
 
 # This is a unified interface for stochastic variational inference in Pyro.
-# The actual construction of the loss is taken care of by `loss`.
+# The actual construction of the loss is taken care of by `loss_factory`.
 # See http://docs.pyro.ai/en/stable/inference_algos.html
 class SVI:
-    def __init__(self, model, guide, optim, loss):
+    def __init__(self, model, guide, optim, loss_factory):
```

Comment:
I think we should keep the original name `loss` so as to support specification as kwarg. It is common to pass this as a kwarg as in `SVI(model, guide, optim=optim, loss=loss)`. I agree that `loss` is a confusing name for what is actually a factory or loss function, but OTOH I'd prefer not to break existing code and idioms.